lib/deploy/activate.rb

```diff
@@ -61,7 +61,8 @@ def setup_idp_config_symlinks
 
       files_to_link.each do |file|
         symlink_verbose(
-          File.join(root, idp_config_checkout_name, "#{file}.yml"),
+          # File.join(root, idp_config_checkout_name, "#{file}.yml"),
```

Comment:
This change needs to be reverted and all of the temporary data files removed before merging. This is just so that we're not publishing actual data live on the internet in my personal sandbox environment during development/testing.

Reply:
moved this to a separate commit on my personal sandbox branch